renovate.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Renovate configuration for WAYSON",
+  "extends": [
+    "config:base"
+  ],
+  "schedule": [
+    "before 9am on Monday"
+  ],
+  "packageRules": [
+    {
+      "matchManagers": [
+        "github-actions"
+      ],
+      "groupName": "GitHub Actions",
+      "addLabels": [
+        "actions"
+      ]
+    },
+    {
+      "matchManagers": [
+        "docker-compose"
```

Comment:
Since this project isn't set up with a docker-compose, I suspect we don't need this group!

Reply:
thanks, removed docker group